feathr_project/feathr/api/app/backends/purview_backend/purview_backend.py

```diff
@@ -0,0 +1,139 @@
+import configparser
+from sys import getrefcount
+from app.backends.backend_base import FeathrBackendAbstract
+from typing import Optional
+from app.core.error_code import ErrorCode
+from feathr.constants import *
+
+from app.core.feathr_api_exception import FeathrApiException
+from app.backends.purview_backend.purview_backend_config import config_path,getRegistryClient
+
+
+DEFAULT_PURVIEW_NAME = "feathrazuretest3-purview1"
```

Comment:
this should always come from yaml config and we should not hard code our test env stuff here.

Reply:
@blrchen can we merge these into config yaml in docker deployment?